features/liquidity/components/BondLiquidityDialog.tsx

```diff
@@ -56,17 +55,15 @@ export const BondLiquidityDialog = ({
       ? liquidity?.fluid.provided.tokenAmount ?? 0
       : liquidity?.staked.provided.tokenAmount ?? 0
 
-  const [tokenAmount, setTokenAmount] = useState(0)
+  const maxDollarValueLiquidity =
+    dialogState === 'stake'
+      ? liquidity?.fluid.provided.dollarValue ?? 0
+      : liquidity?.staked.provided.dollarValue ?? 0
 
-  const [maxDollarValueLiquidity] = usePoolTokensDollarValue({
-    poolId,
-    tokenAmountInMicroDenom: maxLiquidityTokenAmount,
-  })
+  const [tokenAmount, setTokenAmount] = useState(0)
 
-  const [liquidityDollarAmount] = usePoolTokensDollarValue({
-    poolId,
-    tokenAmountInMicroDenom: tokenAmount,
-  })
+  const liquidityDollarAmount =
+    (tokenAmount / maxLiquidityTokenAmount) * maxDollarValueLiquidity
```

Comment:
what does maxLiquidityTokenAmount mean? Maybe change to totalLiquidityTokenAmount and totalLiquidityDollarValue? 

I think making names more clear will help us onboard more devs.